src/services/mail-service.ts

```diff
@@ -0,0 +1,32 @@
+import nodemailer from "nodemailer";
+import config from "../config/config"
+
+
+export const getEmailService = async () => {
+    try {
+        const transporter = nodemailer.createTransport(config.smtp);
+
+
+          const info = await transporter.sendMail({
+            from: config.from_email,
+            to: "syed.yarooq@gmail.com",
+            subject: "Hello from Yarooq",
+            text: "Hello Yarooq?",
+            html: "<strong>Hello Yarooq?</strong>",
+            headers: { 'x-myheader': 'test header' }
+          });
```

Comment:
The content has to be get by the other sql commands.

Reply:
yes no worries work is in progress Once It complete I change The Status of this PR to done